### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by mwesigwa**
Due to popular demand, this PR introduces the long anticipated Rubocop gem to Landable. :) The current config is pretty loose but we can tighten that up as the team gets used to the new Sheriff in town.

### CODE REVIEW

File: db/migrate/20130909182715_landable_page_revisions__break_out_snapshot.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 require_dependency 'landable/page_revision'
 class LandablePageRevisionsBreakOutSnapshot < Landable::Migration
-
-  class Landable::PageRevision < ActiveRecord::Base
-    store :snapshot_attributes, accessors: [:body]
+  class Landable
+    class PageRevision < ActiveRecord::Base
+      store :snapshot_attributes, accessors: [:body]
+    end
```
Comment (dkaplan88) [inline db/migrate/20130909182715_landable_page_revisions__break_out_snapshot.rb]: I think the spec may be failing because of the closing of this class...

**Comment by mwesigwa** [comment]
@dkaplan88 I got this PR to pass so please take a look at it. Thanks..

File: app/controllers/landable/public/sitemap_controller.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ def index
           host: Landable.configuration.sitemap_host || request.host,
           protocol: Landable.configuration.sitemap_protocol,
           exclude_categories: Landable.configuration.sitemap_exclude_categories,
-          sitemap_additional_paths: Landable.configuration.sitemap_additional_paths,
```
Comment (dkaplan88) [inline app/controllers/landable/public/sitemap_controller.rb]: BOO RuboCop, I am a fan of trailing commas as it makes it easier whenever you need to add on later

**Comment by dkaplan88** [comment]
just merged in another PR, so we need to fix the merge conflict

**Comment by mwesigwa** [comment]
@dkaplan88 I removed the redundant checks.

**Comment by dkaplan88** [comment]
@mwesigwa It looks like Travis Failed bc of Rubocop, we probably want this as a silent failure in Travis? 

image

**Comment by mwesigwa** [comment]
I think the whole point of Rubocop is to fail loudly. That way we can force the committer to fix their code before merging it in.. I'm the sufferer right now though but I think it's worth the pain.

**Comment by dkaplan88** [comment]
OK that works for me!

**Comment by dkaplan88** [comment]
Looks Fantastic!! This is awesome! :+1:

**Pull request merged**